hw/ip/rv_core_ibex/data/rv_core_ibex.hjson

```diff
@@ -671,6 +671,38 @@
           },
         ]
       },
+
+      { name: "RND_DATA",
+        desc: "Random data from EDN",
+        swaccess: "ro",
+        hwaccess: "hwo",
+        fields: [
+          { bits: "31:0",
+            name: "DATA",
+            resval: "0x0",
+            desc:  '''
+              Random bits taken from the EDN. !!RND_STATUS indicates if this
+              data is valid. When valid, reading from this register invalidates
+              the data and requests new data from the EDN. The register becomes
+              valid again when the EDN provides new data.  When invalid the
+              register value will read as 0x0.
+            '''
+          },
+        ]
+      },
+
+      { name: "RND_STATUS",
+        desc: "Status of random data in !!RND_DATA",
+        swaccess: "ro",
+        hwaccess: "hrw",
+        resval: "0",
+        fields: [
```

Comment:
Is there a naturally/easily available bit that we could put in here alongside RND_DATA_VALID that says "we have a pending request for data from the EDN" ?  That might resolve the boot-time validity concern that was discussed by @tjaychen and @rswarbrick (?)

Or does that make things too complicated?

Reply:
A pending request bit is easy enough. No harm in including it even if software doesn't bother to use it in the end.

Reply:
... says the designer! :-) It *is* another thing to verify. But not a big thing and I think we're just doing some half-hearted system-level verification for this stuff anyway.